src/woo_publications/publications/tests/test_documents_admin.py

```diff
@@ -614,3 +614,30 @@ def test_bulk_removal_action(self, remove_from_index_by_uuid_delay: MagicMock):
             remove_from_index_by_uuid_delay.assert_any_call(
                 model_name="Document", uuid=doc_uuid, force=True
             )
+
+    @patch("woo_publications.publications.admin.remove_document_from_index.delay")
+    def test_publication_revoke_instance_action(
+        self,
+        mock_remove_document_from_index_delay: MagicMock,
+    ):
+        DocumentFactory.create(publicatiestatus=PublicationStatusOptions.published)
+        DocumentFactory.create(publicatiestatus=PublicationStatusOptions.published)
+        DocumentFactory.create(publicatiestatus=PublicationStatusOptions.revoked)
```

Comment:
should one be concept instead of published?

Reply:
Yes, if I can remember properly I copy and pasted this from another test. I will look into this to see if we have this also in different tests.

Reply:
https://github.com/GPP-Woo/GPP-publicatiebank/pull/265 I think this should fix the existing ones as well.